ts/screens/wallet/payment/PickPspScreen.tsx

```diff
@@ -102,66 +104,98 @@ class PickPspScreen extends React.Component<Props> {
     this.props.loadAllPsp(idWallet, idPayment);
   }
 
+  private headerItem = (
+    <View style={styles.padded}>
+      <View style={styles.line1}>
+        <LabelSmall weight="Regular" color="bluegrey">
+          {I18n.t("wallet.pickPsp.provider")}
+        </LabelSmall>
+        <LabelSmall weight="Regular" color="bluegrey">
+          {`${I18n.t("wallet.pickPsp.maxFee")} (€)`}
+        </LabelSmall>
+      </View>
+      <View spacer />
+      <ItemSeparatorComponent noPadded />
+    </View>
+  );
+
   private getListItem = (psp: ListRenderItemInfo<Psp>) => {
     const { item } = psp;
+
     return (
       <TouchableDefaultOpacity
         onPress={() => this.props.pickPsp(item.id, this.props.allPsps)}
         style={styles.itemContainer}
       >
         <View style={styles.line1}>
-          <Image
-            style={styles.flexStart}
-            resizeMode={"contain"}
-            source={{ uri: item.logoPSP }}
-          />
-          <IconFont
-            name={"io-right"}
-            size={ICON_SIZE}
-            color={customVariables.contentPrimaryBackground}
-          />
+          {!this.state.hasImageLoadingError ? (
```

Comment:
There is a problem in the way you check if an image has an error.

The state is the same for all the items you generate, so if 1 of the images is not reachable also all the others doesn't show the image.

What about using the `useImageResize` utils:
```suggestion
         {useImageResize(BASE_IMG_W, BASE_IMG_H, item.logoPSP).fold(
          <Body>{item.businessName}</Body>,
          size => {
            const imageStyle: StyleProp<ImageStyle> = {
              width: size[0],
              height: size[1],
              resizeMode: "contain"
            };
            return (
              <Image
                style={imageStyle}
                resizeMode="contain"
                source={{ uri: item.logoPSP }}
              />
            );
          }
        )}
```
Of course being a hook you should convert the in component in a function component.